### DISCUSSION TITLE
Commands are not working in groups

### DISCUSSION THREAD

**Discussion by amirHossein5** [nutgram/nutgram#776] [Q&A]
I'm using laravel, and I want to specify set of commands that an administrator of the group or channel can send to bot inside of that group, so:

```php
$bot->onCommand('some', function (Nutgram $bot) {
    $bot->sendMessage('received');
})->description('some')
    ->scope(new BotCommandScopeAllChatAdministrators());
```

and when I type /some or /some@my_bot_id or @my_bot_id/some it doesn't work.

I've also saw https://github.com/nutgram/nutgram/issues/717, and in config/nutgram.php I've set:

```php
    'config' => [
        'botName' => 'my_bot',
    ],
```

**Comment by Lukasss93** [comment]
You need to set:
```php
'bot_name' => 'my_bot'
``` 

![immagine](https://github.com/user-attachments/assets/911a9c36-a4ec-4225-8803-a090cdc71fa5)

**Reply by amirHossein5** [reply]
Yeah that works thanks, also the name is confusing, do botName means the bot name or id, e.g, example bot as name, or example_bot as id? I've used bot id.

**Reply by amirHossein5** [reply]
But I've faced another problem, the command is only available in groups not channels(even if I copy the message text from group and send it in channel)

**Reply by amirHossein5** [reply]
@Lukasss93

**Reply by Lukasss93** [reply]
Telegram works like this, it doesn't show you the list of commands

**Reply by amirHossein5** [reply]
But also the framework doesn't respond to them. 

Now I'm reading messages inside of fallback (they go to fallback) and check the text, but for some reason the user is null, so I can't check for user that is admin or not.

**Reply by amirHossein5** [reply]
I want to have a command that admins can get their chat id (channel or group), and save them in admin panel for further use. But I couldn't find a way that they can directly message the bot and bot print list of the chats that is part of. So I chose this direction.
